app/screens/sso/sso.js

```diff
@@ -85,6 +95,25 @@ class SSO extends PureComponent {
         });
     };
 
+    onMessage = (event) => {
```

Comment:
I'm trying to understand this. So the WebView loads the page, runs `window.postMessage(document.body.innerText)`, and then we receive the contents of the page in `event.nativeEvent.data` so that we can display the error?